FrameworkSystem/DB/ProxyDB.py

```diff
@@ -75,6 +90,16 @@ def __initializeDB(self):
                                      'PrimaryKey': 'Id'
                                      }
 
+    if 'ProxyDB_CleanProxies' not in tablesInDB:
+      tablesD['ProxyDB_CleanProxies'] = {'Fields': {'UserName': 'VARCHAR(64) NOT NULL',
+                                                    'UserDN': 'VARCHAR(255) NOT NULL',
+                                                    'ProxyProvider': 'VARCHAR(64) DEFAULT "Certificate"',
+                                                    'Pem': 'BLOB',
+                                                    'ExpirationTime': 'DATETIME',
+                                                    },
+                                         'PrimaryKey': ['UserDN', 'ProxyProvider']
+                                         }
+    # FIXME: This table not need in v7 if ProxyDB_CleanProxies will be used
```

Comment:
again, other VO may still want to use the groups, so how would that work ?

Reply:
If in ProxyDB_Proxies stored proxies with group extension it will use, but new proxies will upload without extensions